### ISSUE TITLE
Merge pelita_template back in to the core

### ISSUE THREAD

**Issue by otizonaizit** [ASPP/pelita#692]
Now that the API rewrite is ready, I'd like to move the demos in https://github.com/ASPP/pelita_template back to the core. This makes cross-references and testing easier. The https://github.com/ASPP/pelita_template/blob/master/README.md file should become the core of the new documentation.

Note that this is not a problem for the group repos: we initialize them anyway by just copying the content of `pelita_template`. We can as well copy the contents of `pelita/demos`.

@keszybz @Debilski : what do you think?

**Comment by Debilski** [comment]
Makes sense. I am not 100% sure, it must be the `README.md`, as people are usually looking at it with a browser anyway, so we could as well make it a proper webpage again. (And have a proper index always visible so that no-one has to scroll up and down in order to find anything.)

**Comment by otizonaizit** [comment]
> we could as well make it a proper webpage again

Sure, that is what I meant when I wrote "should become the core of the new documentation". Using the README.md for this was just a shortcut to keep the students from reading the outdated pelita docs.

**Comment by keszybz** [comment]
+1

There can be a readme in a subfolder. Github will display it the same.

**Comment by Debilski** [comment]
I’d suggest we merge it with the complete history (rebased into a folder `template/`?), right? I can try to prepare a PR.

**Comment by Debilski** [comment]
We kind of forgot about this issue. Should I proceed with a new PR on this before our next meeting and merge the two repos?

With GitHub actions we might also be able to have a dummy read-only pelita_template repository that contains the template code and gets updated automatically when pelita-main is updated?

**Comment by otizonaizit** [comment]
Let's talk about this at the meeting. I am not sure it is a good idea for now to merge the repos.
